docs/reference/beta-features.rst

```diff
@@ -46,8 +46,8 @@ The captured graph can be launched as shown below:
       CHECK_HIP_ERROR(hipGraphInstantiate(&instance, graph, NULL, NULL, 0));
       CHECK_HIP_ERROR(hipGraphLaunch(instance, stream));
 
-Graph support requires asynchronous HIP APIs, so users must enable stream-order memory allocation.
-For more details, see :ref:`stream order alloc`.
+Graph support requires asynchronous HIP APIs, so users must use rocBLAS_managed (default) which uses stream-order memory allocation.
```

Comment:
Unclear  ' rocBLAS_managed ' is this a memory type or API?   Can you spell out what API to use or not use for this.